### DISCUSSION TITLE
Move to home after deletion

### DISCUSSION THREAD

**Discussion by never-failing** [simple-login/app#1516] [Q&A]
If I remove an alias from 2 or more pages, after removal it moves to the main page. What can you do to stay where you are? I can’t do it myself, because no access to the code, I use it on a paid basis.

Если я удаляю алиас со 2-й и более страниц, после удаления перемещает на главную страницу. Можно сделать чтоб оставаться там где находишься? Сам не могу сделать, т.к. нет доступа к коду, пользуюсь на платной основе.

**Comment by nguyenkims** [comment]
@never-failing I'm not sure to understand the issue, are you referring to the web app, browser extension, ios or android app?

**Reply by never-failing** [reply]
Web browser. The essence of the question is that when I delete an alias on the 2nd or more pages, after deletion it redirects to the first one.  Those.  if I delete on the 2nd page (paginator number 2), can I stay there after deletion? When there are a lot of aliases to delete, it is not convenient to go to the right page every time.  I'm not sure that you will understand me because of the language barrier.

**Reply by nguyenkims** [reply]
This is indeed a bug, thanks for reporting it! It should be fixed in https://github.com/simple-login/app/pull/1546
